src/raft.c

```diff
@@ -210,7 +204,7 @@ static void executeLogEntry(RedisRaftCtx *rr, raft_entry_t *entry, raft_index_t
     rr->snapshot_info.last_applied_term = entry->term;
     rr->snapshot_info.last_applied_idx = entry_idx;
 
-    RaftRedisCommandArrayFree(&entry_cmds);
+    RaftRedisCommandArrayFree(arr);
```

Comment:
this is a really nitty nit, but at first glance I thought arr was &rr as we use rr a lot.

Reply:
okay, renamed as "array"